### PR TITLE
6 get api reviews reviewid comments

### PR DESCRIPTION
**Pull request by johnnyfwk**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -67,4 +67,49 @@ describe("GET /api/reviews", () => {
...
+                            created_at: expect.any(String),
+                            author: expect.any(String),
+                            body: expect.any(String),
+                            review_id: expect.any(Number)
+                        })
+                    )
+                })
+            })
+    })
+
```
Comment (B7Bprog) [inline __tests__/app.test.js:95]: Please write another test where the id exists in the database, but there are no comments, so expect an empty array and a status 200.

**Review by B7Bprog — CHANGES_REQUESTED** [review]
Hey Johnny, nice work so far, but please check my comment and add another test, then send us a pull request with it.

File: __tests__/app.test.js [hunk 2]
```diff
@@ -67,4 +67,59 @@ describe("GET /api/reviews", () => {
...
+})
+
+describe("GET /api/reviews/:review_id/comments", () => {
+    test("Returns a 200 status code and an array of comments associated with a review ID.", () => {
+        const reviewId = 2;
+        return request(app)
+            .get(`/api/reviews/${reviewId}/comments`)
+            .expect(200)
+            .then((response) => {
+                const commentsById = response.body.commentsById;
```
Comment (B7Bprog) [inline __tests__/app.test.js:79]: Please add a check for the length of the 'commentsById' array. I forgot to mention this previously.

**Review by B7Bprog — CHANGES_REQUESTED** [review]
Please make that little but important change (see comment: checking the comments array's length), then when it's done, feel free to merge, don't need to send another pr for this ticket. 👍
